.github/dependabot.yml

```
@@ -0,0 +1,6 @@
+version: 2
+updates:
```

```suggestion
updates:
  # Before applying suggested PRs, make sure that the new versions of any updated actions are allowed in https://github.com/organizations/geany/settings/actions. Versions are pinned and restricted for security reasons.
```

Can we maybe add the above comment, so that at least in the Dependabot job description we have a hint that versions of the allowed Actions are pinned in the organization settings.

---

Certainly! Done.